deploy.sh

```diff
@@ -75,7 +75,7 @@ echo "Building gh-pages branch for $BRANCH"
 if [[ ! -e "$MANIFEST" ]]; then  npm run build || exit 1; else echo "Already built"; fi
```

Comment:
True for the location.

The check is for use in local mode: you build and run `./deploy.sh` which should work for quick local deploys.